README.md

```diff
@@ -74,15 +74,14 @@ You should create a new user account to whom the virtual mailboxes will belong.
 
 ##### MySQL:
 This distribution contains a file "vexim2/setup/mysql.sql". This file provides the database schema used by vexim. You will have to import it into MySQL, like this:
-
 ```
 # mysql -u root -D YOUR_DATABASE_NAME -p < vexim2/setup/mysql.sql
 ```
-
 Where `YOUR_DATABASE_NAME` is the name of an empty database you have created for vexim. If you want the script to create the database for you and set up access to it, this is also doable: just open it in a text editor, and find a commented out block which begins with `-- CREATE DATABASE` near the top of the file. This block is documented just above it, so you may uncomment it, apply the changes you want and save the file. With the necessary changes made, you should run the following command line to initialize the database:
 ```
 # mysql -u root -p < vexim2/setup/mysql.sql
 ```
+A siteadmin-user is created with an autogenerated password, you will need it for your first login to the vexim-interface.
```

Comment:
OK, I'll suggest some cosmetic changes.
siteadmin-user  → site administrator account OR site administrator user.
the vexim-interface → Vexim or the Vexim interface.